tests/ignite/engine/test_create_supervised.py

```diff
@@ -81,8 +81,8 @@ def _test_create_supervised_trainer(
             else:
                 assert not hasattr(state, "scaler")
     else:
-        if LooseVersion(torch.__version__) >= LooseVersion("1.7.0"):
-            # This is broken in 1.6.0 but will be probably fixed with 1.7.0
+        if LooseVersion(torch.__version__) >= LooseVersion("1.9.0"):
+            # This is broken in 1.8.1 but will be probably fixed with 1.9.0
             with pytest.raises(RuntimeError, match=r"is on CPU, but expected them to be on GPU"):
```

Comment:
@afzal442 to fix the issue we have to update the matched error message:
```diff
- is on CPU, but expected them to be on GPU
+ Expected all tensors to be on the same device
```

Reply:
Oh, thanks. Apologise! I confused then. I made it wrong out of over-smart and overthinking.